src/app/components/faq/faq.js

```diff
@@ -0,0 +1,53 @@
+
+import FaqQuestion from './FaqQuestion/faqQuestion'
+
+export default function Faq() {
+
+    return (
+        <div className="px-8 flex-col items-center text-center ">
```

Comment:
The padding here is misaligned with the stats counter section, take a look at that component to make the padding match

Reply:
done - i think it's right now lol